frontend/templates/product/components/PixelPing.tsx

```diff
@@ -0,0 +1,12 @@
+import { PIXEL_PING_URL } from '@config/env';
+
+export const PixelPing = ({ productcode }: { productcode: string }) => {
+   if (!PIXEL_PING_URL) {
+      return null;
+   }
+   const key = `ifixit/product/${productcode}/en`;
```

Comment:
we may want to specify en-us? this technically isn't all the hits due to shopify stores right?